test/contracts/exec-dest-ctx-child/elrond.json

```diff
@@ -0,0 +1,6 @@
+{
+    "language": "clang",
+    "source_files": [
+        "exec-dest-ctx-child.c"
+    ]
+}
```

Comment:
add empty line?

Reply:
This is how they get generated by erdpy. The change should be in erdpy. I looked into it some time ago, but it was something coming from the depths of python.

Reply:
really annoying when reviewing in github